docs/blogs/links-syphon/syphon.md

```diff
@@ -0,0 +1,151 @@
+# Mixing Metadata, Air and Water: Use the lakeFS Airflow Provider to Link Airflow Execution to lakeFS Data
+
+By [Ariel Shaqed (Scolnicov)]
+
+## Introduction
+
+"How do I integrate X with lakeFS" is an ever-green question on [lakeFS
+Slack][lakefs-slack].  lakeFS takes a "tooling-first" strategy to data
+management: it slots into your existing lineup of tools.  So a significant
+part of our work on lakeFS is devoted to leveraging lakeFS and these other
+tools to improve these integrations.  Our latest addition is Project Syphon!
+
+<img alt="Soda syphon, by Avi Nahmias, available from https://commons.wikimedia.org/wiki/User:Avin" src="https://upload.wikimedia.org/wikipedia/commons/1/1c/Soda_syphon_IMG_1505C.jpg" width=400/>
+
+Airflow was one of our earliest integrations: Itai published the original
+[Air and Water][airflow-air-and-water-blog-1] 2 years ago.  It's time to
+deepen our integrations!  The latest release of the lakeFS provider for
+Airflow, codename "Project Syphon", adds deep linking and metadata between
```

Comment:
This paragraph and the next kind of repeat each other, and neither state the problem being solved - just the solution. 

It's useful to make the article relevant to a wide reader base, and so here I would assume someone know vaguely what Airflow is (and maybe you add a tl;dr above as suggested) - so now I would paint the picture of the current integration limitation/challenge, and explain (and show with screenshot) how good the solution will make people's lives.

*Then* explain what the changes are (deep linking, formatted metadata, etc etc) - but only _after_ explaining to a layperson what the point of the changes are.